### PR TITLE
update export text

### PR DESCRIPTION
**Pull request by DavidDamke**

### CODE REVIEW

File: src/main/resources/layouts/settings.fxml [hunk 1]
```diff
@@ -414,7 +414,7 @@
                                           </HBox>
                                           <HBox prefHeight="42.0" prefWidth="366.0">
                                              <children>
-                                                <Label text="Export your current Data to an Sql script. This is needed if you go from the Snapshot-1.2.0-KeepTimeto the recent release version. " wrapText="true" />
+                                                <Label text="Export your current Data to an Sql script. This is needed if you upgrade to an newer KeepTime version. " wrapText="true" />
                                              </children>
```
Comment (Death111) [inline src/main/resources/layouts/settings.fxml:418]: Could you also check the icon for the export done confirmation?
image
(not related to the issue, but would fit)

**Review by Death111 — CHANGES_REQUESTED** [review]
its good but we may add the missing icon as well

File: src/main/resources/layouts/settings.fxml (hunk 1 above)
Comment (DavidDamke) [reply src/main/resources/layouts/settings.fxml:418]: grafik
Added it :)

**Review by Death111 — APPROVED** [review]

File: src/main/resources/layouts/settings.fxml [hunk 2]
```diff
@@ -414,7 +414,7 @@
                                           </HBox>
                                           <HBox prefHeight="42.0" prefWidth="366.0">
                                              <children>
-                                                <Label text="Export your current Data to an Sql script. This is needed if you go from the Snapshot-1.2.0-KeepTimeto the recent release version. " wrapText="true" />
+                                                <Label text="Export your current Data to an Sql script. This is needed if you upgrade to an newer KeepTime version. " wrapText="true" />
```
Comment (Death111) [inline src/main/resources/layouts/settings.fxml:417]:
```suggestion
                                                <Label text="Export your current Data to an Sql script. This is needed if you upgrade to a newer KeepTime version. " wrapText="true" />
```

**Pull request merged**